src/components/results/dynamic-security-analysis/utils/dynamic-security-analysis-result-utils.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export const dynamicSecurityAnalysisResultInvalidations = ['dynamicSecurityAnalysisResult'];
```

Comment:
move to use-all-computing-status.ts or to dynamic-security-analysis-result-synthesis.tsx ?

Reply:
DONE in https://github.com/gridsuite/gridstudy-app/pull/2499/commits/1ea3c0b9c73d9f412fb210662b2877eb0ddddde1

I have moved all result invalidations of all computations to use-all-computation-status.ts
In the future, should define all updateTypes of notifications as the same in the study-server: https://github.com/gridsuite/study-server/blob/main/src/main/java/org/gridsuite/study/server/notification/NotificationService.java#L55-L79